server/pages/index.js

```diff
@@ -143,7 +143,13 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		store: createReduxStore( initialServerState )
+		store: createReduxStore( initialServerState ),
+		css: new Set(),
```

Comment:
It would be a very bad assumption :) Our CSS depends in a few places on the order loading.